app-frontend/src/app/core/services/project.service.js

```diff
@@ -75,6 +79,26 @@ export default (app) => {
             return this.Project.get({id}).$promise;
         }
 
+        export(projectId, zoom) {
+            return this.userService.getCurrentUser().then(
+                (user) => {
+                    return this.Project.export({
+                        organizationId: user.organizationId,
+                        projectId: projectId,
+                        exportStatus: 'TOBEEXPORTED',
+                        exportType: 'S3',
+                        visibility: 'PRIVATE',
+                        exportOptions: {
+                            rasterSize: zoom
```

Comment:
This should be `resolution` -- but maybe we should just say `zoomLevel` explicitly